<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tabels</class>
 <widget class="QMainWindow" name="Tabels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tabels</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="output1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>421</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxProducts">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show PRODUCTS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCastumers">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show CUSTUMERS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveTableChangesButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>save changes(edit)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>sort list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Put name to search:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelChanges">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit something in the table? Save!</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelSort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show sorted by name table?</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelSavetoFile">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>670</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save information to files?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToTXTButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>690</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToBinButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>690</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.bin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToJsonButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>690</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.json</string>
    </property>
   </widget>
   <widget class="QTableView" name="output1_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>421</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearch_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Put name to search:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelChanges_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit something in the table? Save!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveTableChangesButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>save changes(edit)</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelSort_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show sorted by name table?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>550</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>sort list</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelSavetoFile_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>670</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save information to files?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToTXTButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>690</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToBinButton_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>690</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.bin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToJsonButton_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>690</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.json</string>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>650</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>650</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>610</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>add new product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewForTable">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>620</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSort_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>600</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wanna add item to table?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewForTable_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>620</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>610</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>add new custumer</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSort_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wanna add item to table?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>750</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Accounting for an online hardware store</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Tabels</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>872</x>
     <y>767</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>729</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
